README.md

```diff
@@ -1,3 +1,85 @@
+1. [Configure Docker and Docker Compose for use with Joyent](https://docs.joyent.com/public-cloud/api-access/docker):
+
+```bash
+curl -O https://raw.githubusercontent.com/joyent/sdc-docker/master/tools/sdc-docker-setup.sh && chmod +x sdc-docker-setup.sh
+./sdc-docker-setup.sh -k us-east-1.api.joyent.com <ACCOUNT> ~/.ssh/<PRIVATE_KEY_FILE>
+```
+
+Check that everything is configured correctly by running `./setup.sh`. If it returns without an error you're all set. This script will create and `_env` file that includes the Triton CNS name for the Consul service. You'll want to edit this file to update the username and password for Couchbase.
+
+### Running the cluster
+
+Once you've cloned the repo and run `./setup.sh`, you can start a new cluster with just Docker Compose:
+
+```bash
+$ docker-compose --project-name=cb up -d
+Creating cb_consul_1
+Creating cb_couchbase_1
+```
+
+Because one Couchbase container can get lonely, we can use Docker Compose to give it some friends:
+
+```bash
+$ docker-compose -p cb scale couchbase=3
+Creating couchbase_couchbase_2
+Creating couchbase_couchbase_3
+
+$ docker-compose -p cb ps
+Name                   Command                  State                   Ports
+-------------------------------------------------------------------------------------
+cb_consul_1      /bin/start -server -bootst ...   Up      53/tcp, 53/udp, 8300/tcp...
+cb_couchbase_1   /bin/containerbuddy /usr/...     Up      11207/tcp, 11210/tcp,
+                                                          11211/tcp, 18091/tcp,
+                                                          18092/tcp, 8093/tcp,
+                                                          0.0.0.0:8091/tcp->8091/tcp,
+                                                          0.0.0.0:8092/tcp->8092/tcp,
+cb_couchbase_2   /bin/containerbuddy /usr/...     Up      11207/tcp, 11210/tcp,
+                                                          11211/tcp, 18091/tcp,
+                                                          18092/tcp, 8093/tcp,
+                                                          0.0.0.0:8091/tcp->8091/tcp,
+                                                          0.0.0.0:8092/tcp->8092/tcp,
+cb_couchbase_3   /bin/containerbuddy /usr/...     Up      11207/tcp, 11210/tcp,
+                                                          11211/tcp, 18091/tcp,
+                                                          18092/tcp, 8093/tcp,
+                                                          0.0.0.0:8091/tcp->8091/tcp,
+                                                          0.0.0.0:8092/tcp->8092/tcp,
+```
+
+A shell script (`./demo.sh`) has been provided to run these two commands as well as find and open the Couchbase dashboard in your web browser. Sign in with the username and password you provided in the environment file to see the working cluster. As the cluster scales up you'll be able to see node(s) join the cluster.
+
+### Consul notes
```

Comment:
This section was in the original but I'm not sure why we're including it here rather than in `autopilotpattern/consul`